src/styles/style.css

```diff
@@ -132,22 +133,39 @@ body {
 
 /* Home Tab */
 
-.invest-list-table {
-	display: block;
+.invest-list-group {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
```

Comment:
So this will wrap to make as many columns as needed to fit them all on the screen (which is currently 2 columns)? Do more columns form if you change the window size?

Reply:
Yes, it wraps to a second column if the list is too long vertically (which it always will be). More columns appear if the window height shrinks too much for two columns. And a horizontal scroll appears if window width shrinks too much.

But my aim was for no scrollbars and two columns by default. So the aim is for that to hold true for most screen sizes with the default window & font sizes.